### ISSUE TITLE
Error running hassio ha logs

### ISSUE THREAD

**Issue by ciotlosm** [home-assistant/hassio-cli#82]
Error running `hassio ha logs` during home assistant startup

```
core-ssh:~# hassio -d ha logs
DEBUG [CmdHomeassistant]: action->'logs', endpoint='logs', serverOverride->'', GET->'true', options->'', rawjson->'false', filter->''
DEBUG [ExecCommand]: basepath->'homeassistant', endpoint->'logs', serverOverride->'', get->'true', Options->'', Filter->'', RawJSON->'false'
DEBUG [GenerateURI]: basepath->'homeassistant', endpoint->'logs', serverOverride->''
DEBUG [RestCall]: data->'http://hassio/homeassistant/logs', GET->'true', payload->''
DEBUG [RestCall]: ResponseBody->''
Error decoding json unexpected end of JSON input: 
```

Expected behaviour: 
A more explicit error message that home assistant container is not yet available.

**Comment by ryanm101** [comment]
What steps to reproduce? was HA restarting when you tried the logs or doing any update?

**Comment by frenck** [comment]
This problem appears when HA is not running (and therefore, no data is returned).

**Comment by frenck** [comment]
We need to be able to handle this correctly.

**Comment by mjrider** [comment]
Fixed in #127 
merged in dev in commit fca215d405f3ab342b065d3c23ead9a652c7a466
